.github/workflows/tests.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         uses: ./.github/actions/setup_lfs
 
       - name: Download Lock File
-        run:  wget -q https://raw.githubusercontent.com/tardis-sn/tardis/master/conda-${{ matrix.label }}.lock
+        run:  wget -q https://raw.githubusercontent.com/tardis-sn/tardis/update_lockfiles_2024/conda-${{ matrix.label }}.lock
```

Comment:
This is a temporary change, will undo once ready to merge.

Reply:
This is ready, please undo.